osi_sensorinputconfiguration.proto

```diff
@@ -64,23 +64,31 @@ package osi;
 message SensorInputConfiguration
 {
     // The mounting position of the sensor (origin and orientation of the sensor coordinate system)
+    // given in vehicle coordinates [1].
     //
-    // given in vehicle coordinates: origin is \c Vehicle::reference_point; the orientation is equal to the orientation of the vehicle.
+    // \arg \b x-direction of sensor coordinate system: sensor viewing direction
+    // \arg \b z-direction of sensor coordinate system: sensor (up)
+    // \arg \b y-direction of sensor coordinate system: perpendicular to x and z right hand system
     //
-    // \arg \b x-direction: longitudinal direction & positive in driving direction
-    // \arg \b y-direction: lateral direction & positive to the left when looking orientated as a driver
-    // \arg \b z-direction: perpendicular to x and z right hand system (up)
+    // \par References: 
+    // [1] DIN ISO 8855:2013-11
     //
-    // See https://en.wikipedia.org/wiki/Axes_conventions#/media/File:RPY_angles_of_cars.png
+    // \note The origin of vehicle's coordinate system in world frame is 
+    // ( \c Vehicle::base.position + Inverse_Rotation_yaw_pitch_roll(\c Vehicle::base.orientation) * \c Vehicle::bbcenter_to_rear) .
+    // The orientation of the vehicle's coordinate system is equal to the orientation 
+    // of the vehicle's bounding box \c Vehicle::base.orientation.
     // \note A default position can be provided by the sensor model (e.g. to indicate the position the model was validated for),
     // but this is optional; the environment simulation must provide a valid mounting position (based on the vehicle configuration)
     // when setting the input configuration.
+    //
     optional MountingPosition mounting_position = 1;
 
     // Field of View in horizontal orientation of the sensor
     //
     // This determines the limit of the cone of interest of ground truth
     // that the simulation environment has to provide.
+    // Viewing range: [-field_of_view_horizontal/2, field_of_view_horizontal]
+    // in sensor coordinate system.
```

Comment:
It seems that the range should be defined as:
``[-field_of_view_horizontal/2, field_of_view_horizontal/2]``
Morover, consider adding the following note to avoid confusion as to what horizontal means:
``Note: 'horizontal' does not necessarily mean 'perpendicular to the local gravitational vector', but more generally refers to the left/right viewing range of the sensor.``